frontend/app/simple_page/page.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Image from "next/image";
+import "./style.css"
+
+const SimplePage = ()  => {
```

Comment:
- You can name this as 'Header' or 'HomePage'
- Update folder name 'simple_page' to 'components' so we can add more components under that folder later.